src/components/atm.text.styles.tsx

```diff
@@ -0,0 +1,74 @@
+import styled from 'styled-components';
+import { colors, margins, fontSizes, fontWeights } from 'global-styles';
+
+const handleType = (name: string | undefined) => {
```

Comment:
```suggestion
export type TextType = 'title' | 'normal' | 'label' | 'error' | 'success';
const handleType = (name?: TextType) => {
```
esse `TextType` vem do comentário do `atm.text.component.tsx` 👍